prisma/schema.prisma

```diff
@@ -95,7 +95,7 @@ model Transaction {
   updated_at          DateTime           @default(now()) @updatedAt @db.Timestamp(6)
   hash                String             @db.VarChar
   network_version     Int
-  fee                 Int
+  fee                 Float
```

Comment:
Is there a bigint type we could use?

Reply:
yes but that would require re-writing the code base, `Int` and `Float` both map to number.